server/src/internalClusterTest/java/org/elasticsearch/snapshots/SnapshotStressTestsIT.java

```diff
@@ -216,11 +216,13 @@ private static class TrackedCluster {
 
         static final Logger logger = LogManager.getLogger(TrackedCluster.class);
         static final String CLIENT = "client";
+        static final String NODE_RESTARTER = "node_restarter";
 
         private final ThreadPool threadPool = new TestThreadPool(
             "TrackedCluster",
             // a single thread for "client" activities, to limit the number of activities all starting at once
-            new ScalingExecutorBuilder(CLIENT, 1, 1, TimeValue.ZERO, true, CLIENT)
+            new ScalingExecutorBuilder(CLIENT, 1, 1, TimeValue.ZERO, true, CLIENT),
+            new ScalingExecutorBuilder(NODE_RESTARTER, 1, 5, TimeValue.ZERO, true, NODE_RESTARTER)
```

Comment:
It seems very unlikely that there's more than 1 of these running at a time, I made the thread pool max size 5 just incase.

To make it happen even once you need to run with `-Dtests.nightly=true` and run it repeatedly for a while.

Reply:
I think there can be 1 to 2 restarter tasks
https://github.com/elastic/elasticsearch/blob/05f80ef058327c1a3d8a911b4872022cb3ff3210/server/src/internalClusterTest/java/org/elasticsearch/snapshots/SnapshotStressTestsIT.java#L284

Reply:
I might just leave at 5, seeing as it's just a max anyway?